### ISSUE TITLE
Dependency issue

### ISSUE THREAD

**Issue by ImanMh** [bundle-stats/bundle-stats#225]
I installed this package and just imported it in my code and I'm getting this issue:
```
Error: Cannot find module 'core-js/modules/es.array.unscopables.flat'
```

<details>
<summary>Package.json</summary>
<p>

```json
{
  "dependencies": {
    "express": "^4.17.1",
    "prop-types": "^15.7.2",
    "react": "^16.8.6",
    "react-dom": "^16.8.6",
    "watchpack": "^1.6.0",
    "webpack-dev-middleware": "^3.7.0",
    "webpack-hot-middleware": "^2.25.0"
  },
  "devDependencies": {
    "@babel/core": "^7.4.5",
    "@babel/node": "^7.4.5",
    "@babel/plugin-proposal-class-properties": "^7.4.4",
    "@babel/plugin-syntax-dynamic-import": "^7.2.0",
    "@babel/preset-env": "^7.4.5",
    "@babel/preset-react": "^7.0.0",
    "babel-eslint": "^10.0.2",
    "babel-loader": "^8.0.6",
    "bundle-stats": "^1.2.1",
    "chalk": "^2.4.2",
    "clean-webpack-plugin": "^3.0.0",
    "css-loader": "^3.0.0",
    "eslint": "^5.3.0",
    "eslint-config-airbnb": "^17.1.1",
    "eslint-plugin-import": "^2.18.0",
    "eslint-plugin-jsx-a11y": "^6.2.3",
    "eslint-plugin-react": "^7.14.2",
    "isomorphic-style-loader": "^5.1.0",
    "postcss-loader": "^3.0.0",
    "webpack": "^4.35.0",
    "webpack-assets-manifest": "^3.1.1",
    "webpack-monitor": "^1.0.14",
    "webpack-node-externals": "^1.7.2",
    "webpack-stylish": "^0.1.8",
    "webpackbar": "^3.2.0"
  }
}
```

</p>
</details>

Let me know if you need more details.

**Comment by vio** [comment]
thanks for reporting it @ImanMh!

In theory the problem was fixed by [`1.2.1`](https://github.com/bundle-stats/bundle-stats/commit/620e6fca8fbbedb369bcb8645c22b4c98f6f312a)

can you please confirm that you are using the latest version (`1.2.1`) and let me know what version of node / os are you having?

**Comment by vio** [comment]
Fixed by #226

@ImanMh I was able to replicate the issue with your dependencies. One of the packages was not including correctly `core-js` and was failing when `core-js` was not available at the project level.

Please update to `1.2.2` and feel free to re-open if you still have issues ;)

**Comment by ImanMh** [comment]
No problems @vio, 
I was using 1.2.1 with node v10.16.0 and npm v6.9.0 and had issues how ever **v1.2.2** fixed the issue. thanks for taking care of that so quickly.

**Comment by ImanMh** [comment]
OK, I just updated to **1.2.2** and importing bundle-stats works all fine but when I add it to the webpack plugins I get the following error: 

```
/@bundle-stats/utils/lib/assets/get-assets-metrics.js:24
  }) => items).flat();
               ^

TypeError: Object.values(...).map(...).flat is not a function
```

Here is how my webpack plugins looks like:

```
import { BundleStatsWebpackPlugin } from 'bundle-stats';

... webpack plugins belowo ...

  plugins: [
    new BundleStatsWebpackPlugin(),
  ],
```

**Comment by vio** [comment]
Thanks for checking @imanmh! Probably  core-js is still not working correctly when packed, will have a look.

As a quick workaround, you can try to run node v11

**Comment by ImanMh** [comment]
Thank you, Works perfectly with v.11.15.0 of node.

**Comment by vio** [comment]
babel target was node `current` version, solved by 59d133fd052d01a91b341c9de70d25bbda2fd32d
